mapie/conformity_scores/conformity_scores.py

```diff
@@ -300,13 +367,33 @@ def get_bounds(
             (n_samples, n_alpha).
             - The upper bounds of the prediction intervals of shape
             (n_samples, n_alpha).
+
+        Raises
+        ------
+        ValueError
+            If beta optimisation with symmetrical conformity score function.
         """
+        if self.sym and optimize_beta:
+            raise ValueError(
+                "Beta optimisation cannot be used with "
+                + "symmetrical conformity score function."
+            )
+
         y_pred, y_pred_low, y_pred_up = estimator.predict(X, ensemble)
         signed = -1 if self.sym else 1
 
+        if optimize_beta:
+            beta_np = self._beta_optimize(
+                alpha_np,
+                conformity_scores.reshape(1, -1),
+                conformity_scores.reshape(1, -1),
+            )
+        else:
+            beta_np = alpha_np / 2
+
         if method == "plus":
-            alpha_low = alpha_np if self.sym else alpha_np / 2
-            alpha_up = 1 - alpha_np if self.sym else 1 - alpha_np / 2
+            alpha_low = alpha_np if self.sym else beta_np
```

Comment:
I find it a little weird to call the bound `beta_np`. It really associates it with the beta optimizer. Could we not simply call it the bound?

Reply:
It's a possibility. Up until now, we've always done it that way. This could be the subject of an issue if the internal naming system poses problems in the long term.